### ISSUE TITLE
Nioh: save data upload fails: No save files matching the specified filters have been found

### ISSUE THREAD

**Issue by mmbossoni** [derrod/legendary#533]
## Platform
Operating system and version: Up to date archlinux
Legendary version (`legendary -V`): legendary version "0.20.31", codename "Dark Energy (hotfix #5)"
Running legendary through Heroic

## Expected Behavior
Save upload succeed

## Current Behavior
Save upload fails

## Steps to Reproduce
1. Run Nioh and create a save data
2. Try to upload it

## Additional information
(ommited my egs id and replaced with aaaaaaa)
```
[cli] INFO: Overriding save path with "/mnt/games/marcelo/epic/prefixes/Nioh/pfx/drive_c/users/steamuser/Documents/KoeiTecmo/NIOH/Savedata/aaaaaaa/"...
[cli] INFO: Local save for "Nioh: The Complete Edition" is newer
[cli] INFO: - Cloud save date: 2022-12-17 17:57:52
[cli] INFO: - Local save date: 2023-01-01 19:08:44
[cli] INFO: Uploading local savegame...
[SGH] INFO: Packing savegame for "b5e42b0dc2f544a5bbb4080a433b8b71", input folder: /mnt/games/marcelo/epic/prefixes/Nioh/pfx/drive_c/users/steamuser/Documents/KoeiTecmo/NIOH/Savedata/aaaaaaa/
[SGH] WARNING: No save files matching the specified filters have been found.
[Core] INFO: No files to upload. If you believe this is incorrect run command with "--disable-filters"
```

**Comment by derrod** [comment]
What does the `info` command say for that title? Could just be that the developers specified incorrect filters.

**Comment by mmbossoni** [comment]
```
Game Information:
- App name: b5e42b0dc2f544a5bbb4080a433b8b71
- Title: Nioh: The Complete Edition
- Latest version: Ver1.24.08.57450
- All versions:
 +  Windows : Ver1.24.08.57450
- Cloud saves supported: True
- Cloud save folder (Windows): {UserDir}/KoeiTecmo/NIOH/Savedata/{EpicID}/
- Cloud save folder (Mac): (None)
- Is DLC: False
- Activates on external platform: No
- Extra launch options: (None)
- Owned DLC: (None)

Installation information:
- Platform: Windows
- Version: Ver1.24.08.57450
- Install size: 73.70 GiB
- Install path: /mnt/games/marcelo/epic/Nioh
- Save data path: /mnt/games/marcelo/epic/prefixes/Nioh/pfx/drive_c/users/steamuser/Documents/KoeiTecmo/NIOH/Savedata/75dea970468c42a88514b82b15858dad/
- EGL sync GUID: 
- Install tags: (None, all game data selected for install)
- Requires ownership verification token (DRM): False
- Installed DLC: (None)

Manifest information:
- Manifest size: 4710.5 KiB
- Manifest type: Binary
- Manifest version: 17
- Manifest feature level: 17
- Manifest app name: 9b6429d812dd4300abd63b5ed72e99f8
- Launch EXE: nioh_launcher.exe
- Launch Command:  
- Build version: Ver1.24.08.57450
- Build ID: 5eL1rE56GJiFmuUk81Nw0ey5Fhk
- Prerequisites: (None)
- Install tags: (empty)
- Files: 413
- Chunks: 72855
- Disk size (uncompressed): 73.70 GiB
- Download size (compressed): 43.87 GiB
- Disk size by install tag: N/A
- Download size by install tag: N/A
```

**Comment by mmbossoni** [comment]
Inside save folder
```
.:
SAVEDATA00  SYSTEMSAVEDATA00

./SAVEDATA00:
SAVEDATA.BIN

./SYSTEMSAVEDATA00:
SAVEDATA.BIN
```

**Comment by CommandMC** [comment]
This should be resolved with 175168a

**Comment by derrod** [comment]
Yep, should be fixed now with 0.20.33
